src/main/java/uk/gov/justice/digital/delius/data/api/ContactSummary.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.justice.digital.delius.data.api;
+
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotNull;
+import java.time.OffsetDateTime;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ContactSummary {
+    @NotNull
+    @ApiModelProperty(name = "Contact id", required = true)
+    private Long contactId;
+
+    @ApiModelProperty(name = "Contact start date & time", example = "2021-05-25T10:00:00+01:00")
+    private OffsetDateTime contactStart;
+
+    @ApiModelProperty(name = "Contact end date & time", example = "2021-05-25T11:00:00+01:00")
+    private OffsetDateTime contactEnd;
```

Comment:
With Rajiv's work we continued the pattern of Community API only using LocalDateTimes on it's exposed APIs, using offsets would be a departure from that (although I agree offsets are better).

Reply:
There are still offset date times all over the appointment API, which is why they were used here. Is there work outstanding to convert to locals?

Reply:
OK - Ignore me. I though we were using offsets on DAPI and locals on CAPI but it's the other way around!

Reply:
Is it possible to explain what the issue is with using LocalDateTime? What would be scenario we would get the wrong time given we always run in Europe/London timezone? Are contact times wrong by an hour in summer or something?

Reply:
For appointments we were trying to use an unambiguous time value as far as we could as the data crosses a number of system boundaries. The source system (Interventions) is using UTC and the ultimate destination (Delius) is un-timezoned (!). Relying on the environment timezone of the three intermediate systems seems brittle, especially as they are deployed in different environments, however, I appreciate that this is not something that can be fixed without coordination across teams and using OffsetDateTime is inconsistent with existing CAPI code. Maybe this is something that needs a wider conversation?

Reply:
I remember for Alfresco someone had to use OffsetDateTime which I guess suggests Alfresco was setup differently to Delius.

TBH after all these years timezones *still* confuse me. Which is why I like to see a "test case" to understand the real world issue. And it sounds like this is an issue that *might* happen if Delius was change timezones.

Reply:
Yes timezones are the source of ultimate confusion for me too :) This doesn't fix the overall problem but it does take out one external variable which we saw as a positive. The test case would be an end-to-end test across systems where the timezones of each system env are the same, one is different, two are different etc.